inc/command/class-command.php

```diff
@@ -188,14 +203,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
+		$use_chassis = $input->getOption( 'chassis' );
+		$tests_folder = $module !== 'project' ? "altis/$module/tests" : '../tests';
+		$project_subdomain = $this->get_project_subdomain();
+
+		// Write the default config.
+		$config = [
+			'paths' => [
+				'tests' => $tests_folder,
+				'output' => 'altis/dev-tools/tests/_output',
+				'data' => 'altis/dev-tools/tests/_data',
+				'support' => 'altis/dev-tools/tests/_support',
+			],
+			'actor_suffix' => 'Tester',
+			'extensions' => [
+				'enabled' => [
+					'Codeception\Extension\RunFailed',
+				],
+				'commands' => [
+					'Codeception\Command\GenerateWPUnit',
+					'Codeception\Command\GenerateWPRestApi',
+					'Codeception\Command\GenerateWPRestController',
+					'Codeception\Command\GenerateWPRestPostTypeController',
+					'Codeception\Command\GenerateWPAjax',
+					'Codeception\Command\GenerateWPCanonical',
+					'Codeception\Command\GenerateWPXMLRPC',
+				],
+			],
+			'modules' => [
+				'config' => [
+					'WPDb' => [
+						'dsn' => '%TEST_SITE_DB_DSN%',
+						'user' => '%TEST_SITE_DB_USER%',
+						'password' => '%TEST_SITE_DB_PASSWORD%',
+						'dump' => '%TEST_SITE_DB_DUMP%',
+						'populate' => true,
+						'cleanup' => true,
+						'waitlock' => 10,
+						'url' => '%TEST_SITE_WP_URL%',
+						'urlReplacement' => true,
```

Comment:
```suggestion
						'urlReplacement' => true,
						'originalUrl' => 'https://dev.altis.dev',
```

Should make replacement more accurate.

Reply:
But it'll choke on custom dumps, hence why I preferred the detection method.